### ISSUE TITLE
Chrome version is gone

### ISSUE THREAD

**Issue by PConsoleOtaku** [chenkai2/tumblr_video_downloader#2]
Chrome version is no longer valid (error 404 The requested URL was not found on this server). Please fix this.

**Comment by LenoJames** [comment]
Yes, can confirm. I would like to reinstall. Where is the Chrome version located now?
